code/modules/mob/living/living_defense.dm

```diff
@@ -68,6 +68,19 @@
 		else
 				return 0
 
+/mob/living/proc/set_combat_mode(new_mode, silent)
+	if(combat_mode == new_mode)
+		return
+	combat_mode = new_mode
+	client?.show_popup_menus = !src.combat_mode
```

Comment:
```suggestion
	client?.show_popup_menus = !combat_mode
```

Reply:
This doesnt work, its a byond bug me and I think jared discussed? It's really weird but it tried to find combat_mode on the client and thus runtimes.

Reply:
```suggestion
	if(client) //Due to a BYOND bug, `client?.show_popup_menus = !combat_mode` will runtime, trying to access the client's inexistent combat_mode var.
		client.show_popup_menus = !combat_mode
```
Let's leave that information for future coders.